modules/server/src/main/ts/bedrock/compiler/Imports.ts

```diff
@@ -129,10 +142,15 @@ window.removeEventListener('error', importErrorHandler);
 export {};`;
 };
 
-export const generateImports = (useRequire: boolean, scratchFile: string, srcFiles: string[]): string => {
+export const generateImports = (useRequire: boolean, scratchFile: string, srcFiles: string[], polyfills: string[]): string => {
+  const polyfillPaths: Record<string, string> = {};
+  polyfills.forEach((name) => {
+    polyfillPaths[name] = convertPolyfillNameToPath(name);
+  });
+
   if (hasTs(srcFiles)) {
-    return generateImportsTs(useRequire, scratchFile, srcFiles);
+    return generateImportsTs(useRequire, scratchFile, srcFiles, polyfillPaths);
```

Comment:
Since both functions are called identically, you could refactor this. 

```
const f = hasTs(srcFiles) ? generateImportsTs : generateImportsJs;
f(useRequire, scratchFile, srcFiles, polyfillPaths);
```

Reply:
Good point, I didn't even look at that since that part was done previously (I just was focused on passing the polyfills through).

Reply:
Yeah, it's an aside.

Reply:
Fixed